src/actions/authActions.js

```diff
@@ -50,7 +58,7 @@ const Crashlytics = firebase.crashlytics();
 const storage = Storage.getInstance('db');
 const chat = new ChatService();
 
-export const loginAction = (pin: string) => {
+export const loginAction = (pin: string, successCallback?: Function) => {
```

Comment:
I would rename the `successCallback` to something more manifest, like 'onLoginSuccess'

Reply:
OK.